IdentityCore/src/auth_scheme/MSIDAuthenticationSchemePop.h

```diff
@@ -32,7 +32,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface MSIDAuthenticationSchemePop : MSIDAuthenticationScheme
 
-- (instancetype)initWithSchemeParameters:(NSDictionary *)schemeParameters;
+@property (nonatomic) NSString *kid;
+@property (nonatomic) NSString *req_cnf;
```

Comment:
We can keep these properties private in the implementation file itself. If you need to access them in test files , you can use KVC like [scheme valueForKey:@"kid"] . If you want to move them to the headers, we will need to mark them as (readonly) in header file and redeclare it as (readwrite) in implementation file.

Reply:
Addressed. I changed it back to private properties. Thank you for the suggestion!